packages/rum/src/viewTracker.ts

```diff
@@ -141,10 +145,11 @@ function trackMeasures(lifeCycle: LifeCycle, scheduleViewUpdate: () => void) {
       scheduleViewUpdate()
     }
   })
-  trackEventCounts(lifeCycle, (eventCounts) => {
+  const { reset } = trackEventCounts(lifeCycle, (eventCounts) => {
     viewMeasures = { ...viewMeasures, ...eventCounts }
     scheduleViewUpdate()
   })
+  resetEventCounts = reset
```

Comment:
It could be nicer to manage the reset by splitting trackMeasures into something like `trackTimings` and `trackEventCounts`

Reply:
The issue is, the name `trackEventCounts` is already used. IMO this doesn't matter too much since I'll remove this in the PR I'm working on.

Reply:
what I had in mind was not to create a new trackEventCounts function but to use it in the main function directly.
Since I don't know what the goal and the implications of the new PR is, it is hard to me to see if it will improve this point.

Reply:
Alright.